velox/tpcds/gen/dsdgen/decimal.cpp

```diff
@@ -219,7 +220,7 @@ int dectostr(char* dest, decimal_t* d, DSDGenContext& dsdGenContext) {
     number /= 10;
 
   dest = (char*)malloc(160);
```

Comment:
Looks like a memory leak. Is this freed? Let's use `std::vector<char> dest(160)`

Reply:
Please grep the tpcds code for similar allocations.